trucks.js

```diff
@@ -501,4 +502,168 @@ var foodTrucks = [
     Facebook: 'https://www.facebook.com/fulltilticecreams',
     Twitter: 'https://twitter.com/fticecream'
   }
-];
\ No newline at end of file
+];
+/* eslint-enable max-len */
+
+//function returns true if the currently checked truck is available
+//on the target day
+function filterBySchedule(truck, day) {
+  if (truck.schedule.indexOf(day) !== -1) {
```

Comment:
Since doing an equality check evaluates to true or false, you can return the result of doing this comparison directly rather than adding the if/else structure.